dev_envs/mac-env-setup.md

```diff
@@ -249,14 +249,14 @@ and paste the contents of your clipboard
 There are some additional setup steps to access the
 [COOL (Cloud-Optimized Operations Lab)](https://github.com/cisagov/cool-system/wiki/):
```

Comment:
We should probably update this to mention that these steps are only necessary if you want to manually configure these vs using the Jamf-pushed configuration.

Reply:
added verbatim.

Reply:
Added in [31a9e38](https://github.com/cisagov/development-guide/pull/87/commits/31a9e3882f1ae43807effc191baf76d3c148bbd4).